github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```
@@ -465,8 +465,8 @@ presubmits:
     - release-\d+\.\d+
     annotations:
       fork-per-release: "true"
-    always_run: false
-    optional: true
+    always_run: true
+    optional: false
```

https://prow.apps.ovirt.org/?job=pull-kubevirt-e2e-kind-k8s-1.17.0-ipv6 does not show a green lane yet. Maybe rather start with

```suggestion
    skip_report: true
```

first?

---

Yes, let's skip reporting and see if it stays green first.

---

Changed it to be optional and skip report as requested. Although the failing tests are flaky on other lanes as well.
But I agree, let's see some greens first.